Config refactoring

tjk584 opened the pull request:
Changed master to main, same pull request as here: https://github.com/SuperDARNCanada/borealis_config_files/pull/3

tjk584 committed:
Refactored cly to new config format

tjk584 committed:
Refactored inv to new config format

tjk584 committed:
Added placeholder for backup n200 16

tjk584 committed:
Refactored pgr to new config format

tjk584 committed:
Refactored rkn to new config format

tjk584 committed:
Refactored sas to new config format

tjk584 committed:
Added new changes to config file

Added separate flags for rx, tx, and rx_int

tjk584 committed:
Added new changes to config file

Added separate flags for rx, tx, and rx_int

tjk584 committed:
Added new changes to config file

Added separate flags for rx, tx, and rx_int

tjk584 committed:
Added new changes to config file

Added separate flags for rx, tx, and rx_int

tjk584 committed:
Added new changes to config file

Added separate flags for rx, tx, and rx_int

tjk584 committed:
Removed count parameters

RemingtonRohel commented:
Add main_antenna_count and interferometer_antenna_count parameters to these, then we should merge and pull at sites.

tjk584 committed:
Updated to reflect config parsing changes

Removed main_antennas and interferometer_antennas and re-added count parameters

tjk584 commented:
Updated all config files to the newest format to work with Borealis v0.6
